Jenkinsfile

```diff
@@ -136,7 +136,8 @@ pipeline {
                         }
                     }
                     steps {
-                        sconsBuild clean: "_build.external${arch}"
+                        sconsBuild clean: "_build.external${arch}",
+                                   failure_artifacts: 'config.log-centos7-gcc'
```

Comment:
If we only archive on failure then how do I know what to compare a failed log file to?